cypress/pages/objectsPage.cy.js

```diff
@@ -0,0 +1,12 @@
+export default class ObjectPage {
```

Comment:
if you want to have separate method in `Page Object` than it will be good to move selector here also. So for now it's easier to move this part to test suite